src/services/keystroke.ts

```diff
@@ -187,13 +211,27 @@ class MQNode extends NodeBase {
           ctrlr.aria.queue(
             cursor.selection.join('mathspeak', ' ').trim() + ' selected'
           );
-        else ctrlr.aria.queue('nothing selected');
+        else {
+          const str = 'nothing selected';
+          ctrlr.aria.queue(
+            ctrlr.ariaStringsOverrideMap?.[str]
+              ? ctrlr.ariaStringsOverrideMap[str]
+              : str
+          );
+        }
         break;
 
       case 'Ctrl-Alt-=':
       case 'Ctrl-Alt-Shift-Right': // speak ARIA post label (evaluation or error)
         if (ctrlr.ariaPostLabel.length) ctrlr.aria.queue(ctrlr.ariaPostLabel);
-        else ctrlr.aria.queue('no answer');
+        else {
+          const str = 'no answer';
+          ctrlr.aria.queue(
+            ctrlr.ariaStringsOverrideMap?.[str]
+              ? ctrlr.ariaStringsOverrideMap[str]
+              : str
+          );
+        }
```

Comment:
It seems like a lot of things have to know about `ariaStringsOverrideMap`. It's also a bit awkward that we're digging into `ctrlr.aria` here and then accessing another property on `ctrlr` to get the translated string. Could the controller handle the translation internally, e.g. via an API that looks like:

```
ctrlr.queueAria('no answer');
```

See: [law of Demeter](https://blog.testdouble.com/posts/2022-06-15-law-of-demeter/), [tell, don't ask](https://martinfowler.com/bliki/TellDontAsk.html)

Reply:
I could do a big switch case in aria, but I didn't want to move the logic from where it originally was if I could prevent it.